### PR TITLE
docs-about-sub-pipeline-vars-and-identity

### PR DESCRIPTION
**Pull request by satr**
## What type of PR? (can choose many)
- [x] 🍕 Added/Updated documentation
- [ ] 🎨 Style
- [ ] 🔒 Security

### CODE REVIEW

**Review by Richard87 — COMMENTED** [review]
I still think its a bit confusing, can we start with a section with the simplest possible configuration?

File: public-site/docs/radix-config/index.md [hunk 1]
```diff
@@ -79,16 +79,48 @@ Make sure you never store secrets or confidential information in any intermitent
...
+### `variables`
+```yaml
+spec:
+  build:
+    subPipeline:
+      variables:
+        VAR1: value1
+        VAR2: value2
+```
+`variables` - (optional, available only in [sub-pipelines](/guides/sub-pipeline), they are ignored for sub-pipeline if the [subPipeline](/radix-config/index.md#subpipeline) option exists) environment variables names and values, provided for all build Radix environments in [sub-pipelines](/guides/sub-pipeline). These common environment variables are overridden by environment-specific environment variables with the same names.
```
Comment (Richard87) [inline public-site/docs/radix-config/index.md:98]: I'm not sure what this means?

1. If any sub-pipeline environment variables are set, all _regular_ variables are discarded?
2. Any sub-pipeline environment variables will merge, and override _regular_ variables?

Will regular build secrets and variables be available?

**Review by Richard87 — CHANGES_REQUESTED** [review]
I still think its a bit confusing, can we start with a section with the simplest possible configuration?

File: public-site/docs/radix-config/index.md (hunk 1 above)
Comment (satr) [reply public-site/docs/radix-config/index.md:98]: 1. Yes 
2. Yes
3. Regular build secrets [are available](https://www.radix.equinor.com/guides/sub-pipeline/example-pipeline-with-build-secrets) with `$(radix.build-secrets)`, env vars as described, but it is good point - I'll fix env-vars examples.

Comment (satr) [reply public-site/docs/radix-config/index.md:98]: Fixed

**Review by Richard87 — APPROVED** [review]
Godkjent

**Pull request merged**